test/parallel/test-dotenv-node-options.mjs

```diff
@@ -0,0 +1,88 @@
+import { spawnPromisified } from '../common/index.mjs';
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
+const validEnvFilePath = '../fixtures/dotenv/valid.env';
+const relativePath = '../fixtures/dotenv/node-options.env';
+
+describe('.env supports NODE_OPTIONS', () => {
```

Comment:
@anonrig the last 2 tests feel like they don't fit this `describe`, as they don't check `NODE_OPTIONS` (Consider this as a non-blocking nit 🙂)

Reply:
👍 Updating it. Thanks!